packages/app/src/app/components/Create/CreateBox/CreateBoxForm.tsx

```diff
@@ -93,66 +143,76 @@ export const CreateBoxForm: React.FC<CreateBoxFormProps> = ({
               Visibility
             </Text>
             <Stack direction="vertical" gap={1}>
-              <Radio
-                checked={permission === 0}
-                onChange={() => setPermission(0)}
-                label="Public"
-              />
-              <Radio
-                checked={permission === 1}
-                onChange={() => setPermission(1)}
-                label="Unlisted"
-              />
-              <Radio
-                checked={permission === 2}
-                onChange={() => setPermission(2)}
-                label="Private"
-              />
+              <Select
+                icon={PRIVACY_OPTIONS[permission].icon}
+                defaultValue={permission}
+                onChange={({ target: { value } }) => setPermission(value)}
+              >
+                <option value={0}>{PRIVACY_OPTIONS[0].description}</option>
+                <option value={1}>{PRIVACY_OPTIONS[1].description}</option>
+                <option value={2}>{PRIVACY_OPTIONS[2].description}</option>
+              </Select>
             </Stack>
           </Stack>
 
           <Stack direction="vertical" gap={2}>
             <Text size={3} as="label">
               Open in
             </Text>
-            <Stack direction="vertical" gap={1}>
-              <Radio
-                disabled={disableEditorChange}
-                checked={editor === 'web'}
-                onChange={() => setEditor('web')}
-                label="Web editor"
-              />
-              <Radio
-                disabled={disableEditorChange}
-                checked={editor === 'vscode'}
-                onChange={() => setEditor('vscode')}
-                label="VSCode"
+            {type === 'sandbox' ? (
+              <Input
+                css={{ opacity: 0.7, cursor: 'not-allowed' }}
+                value="CodeSandbox web editor"
+                disabled
               />
-            </Stack>
-            {disableEditorChange && (
+            ) : (
+              <Select
+                icon={EDITOR_ICONS[editor]}
+                defaultValue={editor}
+                onChange={({ target: { value } }) => setEditor(value)}
+              >
+                <option value="csb">CodeSandbox web editor</option>
+                <option value="vscode">
+                  VSCode desktop (Using the CodeSandbox extension)
+                </option>
+              </Select>
+            )}
+
+            {type === 'sandbox' && (
               <Stack gap={1}>
                 <Icon color="#999" name="circleBang" />
                 <Text size={3} variant="muted">
-                  Sandboxes can only be open in the web editor.
+                  Sandboxes can only be open in the web editor.{' '}
```

Comment:
Suggestion:
"Sandboxes can only be edited using the web editor."
or: "Sandboxes can only be opened in the web editor."